doc/open-extensions/muc_light.md

```diff
@@ -932,6 +932,16 @@ The notifications contain a list of items. The item list may be different from t
 
 **Affiliations change request**
 
+Let's assume room `coven` with following members:
+
+* `crone1` - `owner`
+* `hag77` - `member`
+* `hag88` - `member`
+
+`hag66` is not in the room yet.
+
+User `crone1` wants to promote `hag77` to owner status, add `hag66` to a room and kick `hag88` from a room.
```

Comment:
User `crone1` wants to add `hag66` to the room, kick `hag88` out and make `hag77` the room owner,.